### PR TITLE
feat(assets_picker_text_delegate.dart): add Japanese text delegate

### PR DESCRIPTION
**Pull request by KosukeSaigusa**
Add Japanese text delegate.

### CODE REVIEW

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 1]
```diff
@@ -281,3 +281,48 @@ class RussianTextDelegate implements AssetsPickerTextDelegate {
   String durationIndicatorBuilder(Duration duration) =>
       AssetsPickerTextDelegate.defaultDurationIndicatorBuilder(duration);
 }
+
+/// [AssetsPickerTextDelegate] implements with Russian.
+/// Локализация на русский язык.
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:286]: 🤔

**Review by AlexV525 — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just left a nit picking, others look good.

File: lib/src/delegates/assets_picker_text_delegate.dart (hunk 1 above)
Comment (KosukeSaigusa) [reply lib/src/delegates/assets_picker_text_delegate.dart]: Thank you for checking and I'm sorry for my mistake.
I revised and push it again.

**Review by AlexV525 — APPROVED** [review]

**Comment by AlexV525** [comment]
@all-contributors Add @KosukeSaigusa for translations.

**Pull request merged**